src/client/Microsoft.Identity.Client/ApiConfig/AcquireTokenForClientParameterBuilder.cs

```diff
@@ -93,6 +96,7 @@ public AcquireTokenForClientParameterBuilder WithAzureRegion(bool autoDetectRegi
 
             CommonParameters.AddApiTelemetryFeature(ApiTelemetryFeature.WithAzureRegion, autoDetectRegion);
```

Comment:
>CommonParameters.AddApiTelemetryFeature(ApiTelemetryFeature.WithAzureRegion, autoDetectRegion); [](start = 12, length = 95)

Wondering if you should set the regionToUse value in telemetry here as well

Reply:
Added this to telemetry